docs/README.md

```diff
@@ -0,0 +1,59 @@
+## Synopsis
```

Comment:
Small question: since JSDoc automatically builds index.html using a specified README.md, what is the advantage of moving the file into this folder?

(Github automatically renders a top-level README with the repo, and jsdoc is more agnostic about where the README lives)

Reply:
> Any solution is good with me really, just need to double check all the links to make sure they work.

In the end, I think the underlying problem is that we're trying to accommodate at least three separate viewing modes simultaneously. For now I've hard-coded a few GitHub-Pages specific links (arbitrarily picking the website as the primary way people read the docs, and not trying too hard to support the others). Changes pushed:
https://github.com/statgen/gwas-credible-sets/commit/8dec8c5c3cd21317c237a6d4bd9baa838d0e52c4

It seems silly, but things that get rebuilt often- like the `docs` folder- are a recipe for merge conflicts, so keeping that folder simple helps avoid headaches down the line. 

Very open to revisiting making things more elegant post-release, if you think people would prefer to read docs in other than HTML format.